package.json

```diff
@@ -107,6 +107,7 @@
 		"rollup": "^2.35.1",
 		"rollup-plugin-bundle-size": "^1.0.3",
 		"rollup-plugin-postcss": "^4.0.0",
+		"rollup-plugin-swc3": "^0.3.0",
```

Comment:
You can add this config to `package.json` for peer deps so we pin to a particular major version of swc but keep it optional:
```json
"peerDependenciesMeta": {
  "@swc/core": {
    "optional": true
  },
}
```